rskj-core/src/main/java/co/rsk/peg/BridgeSerializationUtils.java

```diff
@@ -743,6 +744,33 @@ public static CoinbaseInformation deserializeCoinbaseInformation(byte[] data) {
         return RLP.encodeList(rlpElements);
     }
 
+    public static FastBridgeFederationInformation deserializeFastBridgeInformation(byte[] data) {
+        if ((data == null) || (data.length == 0)) {
+            throw new RuntimeException("Invalid serialized Fast Bridge Federation: null data or no data to deserialize");
+        }
+
+        RLPList rlpList = (RLPList)RLP.decode2(data).get(0);
```

Comment:
Should we check if `rlpList` is null?

Reply:
decode2 function can returns empty list or a list populates with values. If data is null, then return empty list.